src/lib/gadgets/bit-slices.ts

```diff
@@ -0,0 +1,156 @@
+/**
+ * Gadgets for converting between field elements and bit slices of various lengths
+ */
+import { bigIntToBits } from '../../bindings/crypto/bigint-helpers.js';
+import { Bool } from '../bool.js';
+import { Field } from '../field.js';
+import { UInt8 } from '../int.js';
+import { Provable } from '../provable.js';
+import { chunk } from '../util/arrays.js';
+import { assert, exists } from './common.js';
+import type { Field3 } from './foreign-field.js';
+import { l } from './range-check.js';
+
+export { bytesToWord, wordToBytes, wordsToBytes, bytesToWords, sliceField3 };
+
+// conversion between bytes and multi-byte words
+
+/**
+ * Convert an array of UInt8 to a Field element. Expects little endian representation.
+ */
+function bytesToWord(wordBytes: UInt8[]): Field {
```

Comment:
Suggestion: add `le` in the function name, especially since it's exported, to explicit the little endianness part, like `wordOfLeBytes` (it does not look good with Camel case, though)